core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -1004,9 +1061,9 @@ object ConsumerGroupCommand extends Logging {
     val offsetsOpt = parser.accepts("offsets", OffsetsDoc)
                            .availableIf(describeOpt)
     val stateOpt = parser.accepts("state", StateDoc)
-                         .availableIf(describeOpt)
-
-    parser.mutuallyExclusive(membersOpt, offsetsOpt, stateOpt)
+                         .availableIf(describeOpt, listOpt)
+                         .withOptionalArg()
```

Comment:
Since we have an optional argument, do we need to validate that it is not provided when `--describe` is used?